### PR TITLE
Create factorial1.cpp

### PR DESCRIPTION
**Pull request by Anuliko**

### CODE REVIEW

File: factorial1.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (haykaslanian) [inline factorial1.cpp]: Please, delete empty line

File: factorial1.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#include <iostream>
+
+int main() 
+{
+  int fact = 1;
+  unsigned long long int N{};
+  std::cin >> N;
+  
+  if (N == 0 || N == 1)
```
Comment (haykaslanian) [inline factorial1.cpp]: Add function for factorial calculation

File: factorial1.cpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  fact = 1;
+  
+  else
```
Comment (haykaslanian) [inline factorial1.cpp:6]: there is no need to separate N==0 and N==1 cases

Comment (Anuliko) [reply factorial1.cpp:6]: ok, I just used the math definition

File: factorial1.cpp [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+unsigned long long factorial(unsigned n)
+{
+  if (n <= 1)
+  return 1;
```
Comment (haykaslanian) [inline factorial1.cpp:5]: For better readability, put 'return 1;' in 2 spaces right

File: factorial1.cpp [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  else
```
Comment (haykaslanian) [inline factorial1.cpp:6]: There is no need of 'else' statement

**Pull request merged**